components/unified-economy/unified-points-wallet.tsx

```
@@ -404,110 +410,175 @@ export default function UnifiedPointsWallet({
     ? (wallet?.status_points ?? 0) // Use status points when boosted
     : (wallet?.earned_points ?? 0); // Use actual earned points when not boosted
 
+  // Calculate total campaign credits (memoized for performance)
+  const totalCampaignCredits = useMemo(
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)